src/common/dice.js

```diff
@@ -133,18 +133,24 @@ class DNDBDice {
         }
         this._rolls = [];
     }
-
+    getDieResult () {
```

Comment:
as suggested in the conversation, rename the method into `rollSingleDie` instead.